pydis_site/apps/home/urls.py

```diff
@@ -33,4 +33,5 @@
     path('logout', LogoutView.as_view(), name="logout"),
 
     path('admin/', admin.site.urls),
+    path('resources/', include('pydis_site.apps.resources.urls', namespace="resources")),
```

Comment:
```suggestion
    path('resources/', include('pydis_site.apps.resources.urls')),
```
You don't need to specify `namespace`, here, it's implied by your `app_name` definition in [`urls.py`](https://github.com/python-discord/site/pull/395/files#diff-f283b3ddd40b412c2652ab141e05729891f34c62cd7bb30db8bb52b3845caf4cR5).

(You're obliged to keep it here for distinction, just thought you should know.)